app-text/scdoc/scdoc-1.2.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Standalone tool for generating man pages with a simple syntax"
+HOMEPAGE="https://git.sr.ht/~sircmpwn/scdoc"
+SRC_URI="https://git.sr.ht/~sircmpwn/scdoc/snapshot/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	sed -i 's/-Werror//' Makefile || die 'Failed to remove -Werror'
+
+	sed -i 's/CFLAGS=/CFLAGS+=/' Makefile || die 'Failed to patch makefile'
+
+	default
```

Comment:
can't you replace the `sed`'s with a patch?

Reply:
It's less likely to require changes than a patch in future (and easier to keep note of for me).

Reply:
then please do them after `default`